package.json

```diff
@@ -110,21 +110,21 @@
     "lint-build": "node ./scripts/rollup/validate/index.js",
     "extract-errors": "yarn build --type=dev --extract-errors",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json && node ./scripts/flow/createFlowConfigs.js && node ./scripts/yarn/downloadReactIsForPrettyFormat.js",
-    "debug-test": "yarn test --debug",
+    "debug-test": "yarn test --deprecated 'yarn test --debug'",
     "test": "node ./scripts/jest/jest-cli.js",
     "test-www": "yarn test --release-channel=www-modern",
-    "test-www-variant": "yarn test --release-channel=www-modern --variant",
-    "test-prod-www": "yarn test --prod --release-channel=www-modern",
-    "test-prod-www-variant": "yarn test --prod --release-channel=www-modern --variant",
-    "test-persistent": "yarn test --persistent",
-    "debug-test-persistent": "yarn test --debug --persistent",
-    "test-prod": "yarn test --prod",
-    "debug-test-prod": "yarn test --debug --prod",
-    "test-prod-build": "yarn test --prod --build",
-    "test-build": "yarn test --build",
-    "test-build-prod": "yarn test --build --prod",
+    "test-www-variant": "yarn test --deprecated 'yarn test-www --variant'",
+    "test-prod-www": "yarn test --deprecated 'yarn test-www --prod'",
+    "test-prod-www-variant": "yarn test --deprecated 'yarn test-www --prod --variant'",
+    "test-persistent": "yarn test --deprecated 'yarn test --persistent'",
+    "debug-test-persistent": "yarn test --deprecated 'yarn test --debug --persistent'",
+    "test-prod": "yarn test --deprecated 'yarn test --prod'",
+    "debug-test-prod": "yarn test --deprecated 'yarn test --debug --prod'",
+    "test-prod-build": "yarn test --deprecated 'yarn test --prod --build'",
+    "test-build": "yarn test --deprecated 'yarn test --build'",
+    "test-build-prod": "yarn test --deprecated 'yarn test --build --prod'",
     "test-build-devtools": "yarn test --build --project devtools",
-    "debug-test-build-devtools": "yarn test --debug --build --project devtools",
+    "debug-test-build-devtools": "yarn test --deprecated 'yarn test-build-devtools --debug'",
```

Comment:
In the end we'll have three base test commands:

- test
- test-www
- test-build-devtools

Reply:
Well, we _already_ only have three commands since the aliases proxy to them.